content/en/docs/v3.6/op-guide/security.md

```diff
@@ -42,8 +42,16 @@ The peer options work the same way as the client-to-server options:
 
 If either a client-to-server or peer certificate is supplied the key must also be set. All of these configuration options are also available through the environment variables, `ETCD_CA_FILE`, `ETCD_PEER_CA_FILE` and so on.
 
+**Common options:**
+
 `--cipher-suites`: Comma-separated list of supported TLS cipher suites between server/client and peers (empty will be auto-populated by Go). Available from v3.2.22+, v3.3.7+, and v3.4+.
```

Comment:
We don't support 3.2 and 3.3, so I think `Available from v3.2.22+, v3.3.7+, and v3.4+.` is not needed.